### DISCUSSION TITLE
[Feature] Allow photos from shared albums to be added to user's private albums

### DISCUSSION THREAD

**Discussion by ashleycollinge1** [immich-app/immich#8423] [Feature Request]
### The feature

I often share albums with my family and they want to have their own separation of photos in their own accounts, like sharing a 'raws' album, and they pick from there to add to their own album. Currently you can only download, like and comment on the picture.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #5359
